lib/money-rails.rb

```diff
@@ -5,6 +5,7 @@
 require "money-rails/money"
 require "money-rails/version"
 require 'money-rails/hooks'
+require 'money-rails/errors'
```

Comment:
Yeah, I thought of that, but this would be a place to put other errors in as well… similar to https://github.com/RubyMoney/monetize/blob/master/lib/monetize/errors.rb. Sounds sane?